pkg/api/meta/deletion.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package meta
+
+import (
+	"errors"
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	// ErrNoDeletionTimestamp is the error returned by GetDeletionTimestamp when there is no
+	// deletion timestamp set on the object
+	ErrNoDeletionTimestamp = errors.New("no deletion timestamp set")
+)
+
+// GetAccessor returns a MetadataAccessor to fetch general information on metadata of
+// runtime.Object types
+func GetAccessor() meta.MetadataAccessor {
+	return accessor
+}
```

Comment:
Just in terms of how we organize things, since this doesn't look delete-specific, might it be better to move this function to `init.go` where `accessor` is actually defined?